### PR TITLE
mutate: change calculation of the mutation score by not counting

### PR DESCRIPTION
**Pull request by joakim-brannstrom**
suppressed as killed.

### CODE REVIEW

**Comment by joakim-brannstrom** [comment]
@peatersson @christoffer-nylen , note in the test cases how the mutation score changed.
Also note how I changed the score calculation.

File: plugin/mutate/doc/design/usability/report.md [hunk 1]
```diff
@@ -507,7 +507,27 @@ There is a psychology game to play with the user here when it comes to visualizi
 partof: REQ-suppress_mutants
 ###
 
-The plugin shall count suppressed mutants as killed when calculating the mutation score.
+The plugin shall **not** count suppressed mutants as killed when calculating
```
Comment (christoffer-nylen) [inline plugin/mutate/doc/design/usability/report.md]:
```suggestion
The plugin shall **ignore** suppressed mutants when calculating
```

File: plugin/mutate/doc/design/usability/report.md [hunk 2]
```diff
@@ -507,7 +507,27 @@ There is a psychology game to play with the user here when it comes to visualizi
...
+the mutation score.
+
+## Why?
+
+Previously the suppressed mutants where counted as killed. After discussing the
```
Comment (christoffer-nylen) [inline plugin/mutate/doc/design/usability/report.md]:
```suggestion
```

File: plugin/mutate/doc/design/usability/report.md [hunk 3]
```diff
@@ -507,7 +507,27 @@ There is a psychology game to play with the user here when it comes to visualizi
...
+matter we changed direction. This is because a suppressed mutant is an
```
Comment (christoffer-nylen) [inline plugin/mutate/doc/design/usability/report.md]:
```suggestion
A suppressed mutant is an
```

File: plugin/mutate/doc/design/usability/report.md [hunk 4]
```diff
@@ -507,7 +507,27 @@ There is a psychology game to play with the user here when it comes to visualizi
...
+
+An *unproductive* mutant encompasses two things. It is either:
+
+ * never intended to be killed because the mutant is *bad*. The tool should,
+   according to the user, not produce them.
+ * the user *kills* the mutant by inspecting that it is OK as is but the user
+   do not want to write a test case to kill it.
+
+It isn't possible to distinguish these two cases from each other without
+further annotations which would complicate the tool unnecessarily. By changing
```
Comment (christoffer-nylen) [inline plugin/mutate/doc/design/usability/report.md]: Good motivation!

**Comment by joakim-brannstrom-work1** [comment]
I am closing this PR so I can open with my work account

**Pull request merged**